contracts/UBI.sol

```diff
@@ -121,6 +134,10 @@ contract UBI is Initializable {
 
     balance[msg.sender] = _initialSupply;
     totalSupply = _initialSupply;
+
+    chainId = _getCurrentChainId();
+    permitTypehash = keccak256("Permit(address owner,address spender,uint256 value,uint256 nonce,uint256 deadline)");
```

Comment:
Shouldn't permit be lowercase for the hash?

Reply:
It shouldn't, it was defined starting with uppercase in the [EIP-2612](https://eips.ethereum.org/EIPS/eip-2612) to also follow [EIP-712](https://eips.ethereum.org/EIPS/eip-712) signature standard.